lib/manager/terraform/lockfile/hash.spec.ts

```diff
@@ -20,14 +20,12 @@ const log = logger.logger as jest.Mocked<Logger>;
 describe(getName(), () => {
   let cacheDir: DirectoryResult;
 
-  beforeAll(async () => {
+  beforeEach(async () => {
     cacheDir = await dir({ unsafeCleanup: true });
     setAdminConfig({ cacheDir: cacheDir.path });
   });
 
-  beforeEach(() => jest.resetAllMocks());
-
-  afterAll(() => cacheDir.cleanup());
+  afterEach(() => cacheDir.cleanup());
```

Comment:
This is the real ci fix. Looks like some tests are iterferring on macos if same files are used